pt/esbocos/originais/sumarizacao-narrativas-web-python.md

```diff
@@ -436,34 +415,35 @@ for period in summ_result["results"]:
 
 # Prémios
 
-O projeto Conta-me Histórias foi o vencedor do [Prémio Arquivo.pt 2018](https://sobre.arquivo.pt/en/arquivo-pt-2018-award-winners/) e o vencedor do [Best Demo Presentation](http://www.ecir2019.org) na [39th European Conference on Information Retrieval (ECIR-19)](http://ecir2019.org/).
+O projeto *Conta-me Histórias* foi o vencedor do [Prémio Arquivo.pt 2018](https://sobre.arquivo.pt/pt/vencedores-premios-arquivo-pt/) e o vencedor da [Best Demo Presentation](http://www.ecir2019.org) na [39th European Conference on Information Retrieval (ECIR-19)](http://ecir2019.org/) (em inglês).
 
 # Financiamento
 
-Ricardo Campos foi financiado pela [ERDF - European Regional Development Fund](https://ec.europa.eu/regional_policy/en/funding/erdf/) através do [Programa Operacional Regional do Norte (NORTE 2020)](https://www.norte2020.pt/), sob o programa Portugal 2020 e fundos nacionais da agência de financiamento à investigacao Portuguesa [Fundação para a Ciência e Tecnologia (FCT)](https://www.fct.pt/) com o projecto PTDC/CCI-COM/31857/2017 (NORTE-01-0145-FEDER-03185). Este financiamento faz parte da linha de pesquisa do projecto 'Text2Story.'
+Ricardo Campos foi financiado pelo [ERDF - European Regional Development Fund](https://ec.europa.eu/regional_policy/en/funding/erdf/) (em inglês) através do [Programa Operacional Regional do Norte (NORTE 2020)](https://www.norte2020.pt/), sob o programa Portugal 2020 e fundos nacionais da agência de financiamento da investigação portuguesa, [Fundação para a Ciência e Tecnologia (FCT)](https://www.fct.pt/) com o projeto PTDC/CCI-COM/31857/2017 (NORTE-01-0145-FEDER-03185). Este financiamento faz parte da linha de pesquisa do projeto 'Text2Story.'
 
 # Bibliografia
 
-* Campos, R., Pasquali, A., Jatowt, A., Mangaravite, V., Jorge, A. (2021). Automatic Generation of Timelines for Past-Web Events. In Gomes, D., Demidova, E., Winters, J., Risse, T. (Eds.), The Past Web: Exploring Web Archives (pp. 225-242). [pdf](https://link.springer.com/chapter/10.1007/978-3-030-63291-5_18)
+* Campos, R., Pasquali, A., Jatowt, A., Mangaravite, V., and Jorge, A.. "Automatic Generation of Timelines for Past-Web Events" In *The Past Web: Exploring Web Archives*, edited by D. Gomes, E. Demidova, J. Winters, and T. Risse, 225-242. Springer: 2021. [https://link.springer.com/chapter/10.1007/978-3-030-63291-5_18](https://link.springer.com/chapter/10.1007/978-3-030-63291-5_18)

```

Comment:
Qual é o padrão de bibliografia utilizado pelo ProgHist? APA? Não achei isso em nossa nossa documentação.

Reply:
Hello @josircg. We ask authors to use [Chicago Manual of Style](https://www.chicagomanualofstyle.org/tools_citationguide.html).

Reply:
Exacto, é o Chicago Manual of Style. Encontra-se referido aqui: [](https://programminghistorian.org/pt/directrizes-autor)